lib/middleman-gh-pages/tasks/gh-pages.rake

```diff
@@ -26,9 +26,9 @@ file GH_PAGES_REF => BUILD_DIR do
       sh "git checkout gh-pages"
     else
       sh "git checkout --orphan gh-pages"
-      sh "touch index.html"
+      FileUtils.touch(File.join(BUILD_DIR, 'index.html'))
```

Comment:
Is the absolute path necessary here considering the current directory is already the `BUILD_DIR`?